molecule/vagrant_packager/box_files/app_metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "fpf/securedrop-app",
+  "description": "This box contains securedrop app server.",
+  "versions": [
+    {
+      "version": "0.5.2",
+      "providers": [
+        {
+          "name": "libvirt",
+          "url": "https://s3.amazonaws.com/securedrop-vagrant/app-staging_0.5.2.box",
+          "checksum_type": "sha256",
+          "checksum": "58f88fbb27b5c73f65340de78c49a0bc36b32408ab4b4cb97868d4ea7beb6743"
+        }
+      ]
+    },
+    {
+      "version": "0.5.2-1",
+      "providers": [
+        {
+          "name": "libvirt",
+          "url": "https://s3.amazonaws.com/securedrop-vagrant/app-staging_0.5.2_1.box",
+          "checksum_type": "sha256",
+          "checksum": "a66128ea3c9212e7f9589f004652d88b003ba4554b5cceec66cd35b3870c9038"
+        }
+      ]
+    }
+  ]
+}
```

Comment:
We should rebuild for 0.6 final now that it's released, no?

Reply:
yeppp going to have to rebuild for that

Reply:
Done.